solana/rpc/websocket.py

```diff
@@ -0,0 +1,162 @@
+"""API WebSocket Client to interact with the Solana JSON RPC Endpoint."""
+from __future__ import annotations
+
+from typing import Union
+
+from solana.publickey import PublicKey
+
+from .providers.ws import Provider
+from .types import RPCMethod, RPCResponse
+
+
+class WebSocketClient:
+    """Websocket client class."""
+
+    def __init__(self, endpoint: str):
+        """Init API client for websocket."""
+        self._provider = Provider(endpoint)
+
+    def account_subscribe(self, pubkey: Union[PublicKey, str]) -> RPCResponse:
```

Comment:
Handle options for `encoding` and `commitment`.

https://docs.solana.com/apps/jsonrpc-api#parameters-44

Reply:
Example: https://github.com/michaelhly/solana-py/blob/0251648328159533eda1bd92a730f13603315936/solana/rpc/api.py#L63